Assets/Scripts/Enemy.cs

```diff
@@ -42,4 +50,9 @@ public float DealDamageToPlayer(float playerHealth)
         return playerHealth;
     }
 
+    public void movementInput()
```

Comment:
Code looks fine. Seems like the issue with the skeleton's animations is in your mecanim set up.